package-lock.json

```diff
@@ -17345,7 +17345,7 @@
         },
         "@duckduckgo/autofill": {
             "version": "git+ssh://git@github.com/duckduckgo/duckduckgo-autofill.git#a4bff09eb0f3b14005c3960c590ec4efb9c5a23c",
```

Comment:
With this line unchanged, does it actually load the correct version when doing `npm ci`?

Reply:
@sammacbeth let me double check, I expected that ref to also change